graphql_compiler/schema_transformation/merge_schemas.py

```diff
@@ -373,33 +418,38 @@ def _process_scalar_definition(scalar, existing_scalars, type_name_to_schema_id,
     # new, valid scalar
     new_definitions = list(merged_schema_ast.definitions)
     new_definitions.append(scalar)
-    new_merged_schema_ast = ast_types.DocumentNode(definitions=new_definitions)
+    new_merged_schema_ast = DocumentNode(definitions=new_definitions)
     new_existing_scalars = set(existing_scalars)
     new_existing_scalars.add(scalar_name)
     return new_existing_scalars, new_merged_schema_ast
 
 
 def _process_generic_type_definition(
-    generic_type, schema_id, existing_scalars, type_name_to_schema_id, merged_schema_ast
-):
+    generic_type: Union[
+        EnumTypeDefinitionNode,
+        InterfaceTypeDefinitionNode,
+        ObjectTypeDefinitionNode,
+        UnionTypeDefinitionNode,
+    ],
+    schema_id: str,
+    existing_scalars: Set[str],
+    type_name_to_schema_id: Dict[str, str],
+    merged_schema_ast: DocumentNode,
+) -> Tuple[Dict[str, str], DocumentNode]:
     """Compare new type against existing scalars and types, update records and schema.
 
     Args:
-        generic_type: Any of EnumTypeDefinitionNode, InterfaceTypeDefinitionNode,
-                      ObjectTypeDefinitionNode, or UnionTypeDefinitionNode, an AST node
-                      representing the definition of a type.
-        schema_id: str, the identifier of the schema that this type came from.
-        existing_scalars: Set[str], set of names of all existing scalars.
-        type_name_to_schema_id: Dict[str, str], mapping names of types to the identifier of the
-                                schema that they came from.
-        merged_schema_ast: DocumentNode, AST representing a schema.
+        generic_type: AST node representing the definition of a type.
+        schema_id: the identifier of the schema that this type came from.
```

Comment:
```suggestion
        schema_id: identifier of the schema that this type came from.
```
Nit: removing article

Reply:
Also shows up in a few places elsewhere, I think ctrl + f ": the" covers all of them